### PR TITLE
[Doc] Publish Data Lakehouse cn doc

### PR DESCRIPTION
**Pull request by amber-create**
Signed-off-by: amber-create <yangyanping@starrocks.com>

Why I'm doing:

What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Review by hellolilyliuyi — DISMISSED** [review]

File: docs/zh/data_source/data_lakes.mdx [hunk 1]
```diff
@@ -2,8 +2,74 @@
...
 
+import DataLakeIntro from '../assets/commonMarkdown/datalakeIntro.md'
+
 # 查询数据湖
 
+<DataLakeIntro />
+
+## 重点概念
+
+- 开源的文件格式
```
Comment (evelynzhaojie) [inline docs/zh/data_source/data_lakes.mdx:13]: 开源 -- 开放的文件格式

File: docs/zh/data_source/data_lakes.mdx [hunk 2]
```diff
@@ -2,8 +2,74 @@
...
+
+---
+
+## Catalog
+
+StarRocks 提供两种类型的 Catalog：Internal Catalog 和 External Catalog。Internal Catalog 用于管理 StarRocks 数据库中存储的数据的元数据。External Catalog 用于连接存储在 Hive、Iceberg、Hudi、Delta Lake 等各种外部数据源中的数据。
+
+## 计算节点
+
+在存算分离架构下，存储和计算的分离降低了扩展的复杂度。StarRocks 计算节点仅存储本地缓存，因此可以根据负载情况灵活地添加或移除计算节点。
```
Comment (evelynzhaojie) [inline docs/zh/data_source/data_lakes.mdx:63]: 因此可以根据 -- 您可以根据

File: docs/zh/data_source/data_lakes.mdx [hunk 3]
```diff
@@ -2,8 +2,74 @@
...
+
+## Data cache
+
+计算节点上的缓存是可选的。在计算节点因为负载变化过快而频繁启动和关闭、或者是查询大多集中在近期数据等场景下，数据缓存意义不大，可以无需开启缓存。
```
Comment (evelynzhaojie) [inline docs/zh/data_source/data_lakes.mdx:67]: 1. 你可以根据实际情况选择打开或者关闭本地缓存。2. 语义：查询大多集中在近期数据等场景下意义不大，这个和存算分离里的说法相反。
image

Comment (evelynzhaojie) [reply docs/zh/data_source/data_lakes.mdx:67]: @EsoragotoSpirit 一起看看

**Review by evelynzhaojie — APPROVED** [review]

**Review by EsoragotoSpirit — APPROVED** [review]

**Pull request merged**